docs/api/api_data.js

```diff
@@ -1,3622 +1,3622 @@
-define({

```

Comment:
Can we get prettier to format this so that the size of this diff is smaller? Thanks :)

Reply:
yeah i'll make a separate PR

Reply:
I made #612 that adds `/docs` to ignores, but I think this is from running `npm run docs`